ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -136,7 +144,17 @@ const EditGasItem = ({ priorityLevel, onClose }) => {
         )}
       </span>
       <span className="edit-gas-item__tooltip">
-        <InfoTooltip position="top" />
+        <EditGasToolTip
+          priorityLevel={priorityLevel}
+          gasContext={{
+            gasLimit,
+            maxFeePerGas,
+            maxPriorityFeePerGas,
+            maxFeePerGasValue,
+            maxPriorityFeePerGasValue,
+            origin,
+          }}
+        />
```

Comment:
we can use `useGasFeeContext` in `EditGasToolTip` to reduce these props to just 2.

Reply:
I thought it would be better to avoid additional calls to `useGasFeeInputs` (which makes network requests in `useGasFeeEstimates`) and since we have all of the data right here we might as well just pass it along?

Reply:
I think including `useGasFeeContext` does not call `useGasFeeInputs` each time. It will only return latest state in the context.

Reply:
Yes, can confirm ☝🏻

Reply:
ah ok nice! 👍

Reply:
I guess the other problem here is that if I use the useGasFeeContext rather than passing along the values, I will have to rerun all [this logic](https://github.com/MetaMask/metamask-extension/blob/f65063ac6ca41bc7ddef78ceda1f6751bc291d3b/ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js#L44) (lines 38 - 64) to get the correct values? cc @jpuri. Am I wrong?

Reply:
@adonesky1 : values used directly from context can be avoided to be passed, for others it is ok to pass I think

Reply:
@jpuri made this change. Let me know what you think.

Reply:
Looks good

Reply:
Oh I found that even this context is not working in tooltips, we need to revert this change :(